src/pages/about-us.js

```diff
@@ -20,6 +28,15 @@ const AboutUs = ({ data: { contentfulAboutUsPage: content } }) => {
   return (
     <Layout>
       <Head page={content} />
+      <AboutUsHero
+        statementText={statementText}
+        supportingStatement1Icon={supportingStatement1Icon}
```

Comment:
wondering if woud make sense to pass just an array of objects `{ icon, text}`, instead of passing six props?

Reply:
Yeah sure. I wondered about that. But then thought maybe it's not worth grouping these separate variables that come from contentful into an array & then taking them back out of the array the other side.

It is less code to change if/when the supporting statements become dynamic, so.... :balance_scale: 

Why not :+1:

Reply:
Mmmm, you're totally right. Now the `AboutUsHero` looks beautiful. Oh `map` :heart_eyes:

Reply:
💯